README.md

```diff
@@ -1,8 +1,138 @@
 # HTML Fundamentals
 
-<iframe width="640" height="480" src="//www.youtube.com/embed/tuDKQxfiXmY?rel=0&modestbranding=1" frameborder="0" allowfullscreen></iframe>
+## Problem Statement
 
-<p><a href="https://www.youtube.com/watch?v=tuDKQxfiXmY">Alternate video link</a>.</p>
+After learning the basics of HTML, including tag and document structure, you
+might be wondering what else is available to use to build out full webpages. How
+do you display an image? How do you add a readable list of topics? How do you
+link page elements beyond text? Now you're ready to explore these fundamental
+tools. Here's a high-level overview before you dive into practicing with the
+labs.
 
+## Objectives
 
-<p data-visibility='hidden'>View <a href='https://learn.co/lessons/html-fundamentals' title='HTML Fundamentals'>HTML Fundamentals</a> on Learn.co and start learning to code for free.</p>
+1. Identify ordered, unordered and definition lists
+2. Identify images
+3. Identify links
+4. Identify HTML validation tools
+
+### Identify ordered, unordered and definition lists
+
+When we want to present a list of items in a clear, readable format, we turn to
+the HTML unordered list, represented by the the `ul` tag.
+
+```
+<ul>
+  <li>One item</li>
+  <li>Another item</li>
+</ul>
+```
+
+If it's important to distinguish a particular order of the items (as for a
+recipe or ranking), we use an ordered list, or the `ol` tag.
+
+```
+<ol>
+  <li>First item</li>
+  <li>Second item</li>
+</ol>
+```
+
+Notice the nesting of our items within the lists. Each `li` is a list item
+contained in the larger `ul` or `ol` container.
+
+Another type of list we can use is a definition list, which defines specific
+types of items.
+
+```
+<dl>
+  <dt>First term</dt>
+  <dd>Term definition</dd>
+</ol>
+```
+
+### Identify images
+
+To include an image in our page, we use an `img` tag.
+
+```
+<img src="myimage.jpg" alt="Alternative Text" title="Display Title"
+width="800" height="600">
+```
+
+There are two noteable things about the `img` tag. The first is that it has no
+closing tag. This tag closes itself. Secondly, it handles a lot of attributes.
+Our `alt` attribute provides descriptive text the browser can display if it
+can't find the image file. The browser can also display the `title` text to give
+the user more information about the image. The `width` and `height` attributes
+define the size of the image that shows up in the browser. 
+
+### Identify links
+
+You might be familiar with basic link structure already, but here are other ways
+we can power them up.
+
+We begin with a standard text hyperlink.
+
+```
+<a href="http://example.com/">This is a link</a>
+```
+
+What if we want to link an image instead of text? We can replace the text within
+the `a` tags with our image tag.
+
+```
+<a href="http://example.com/">
+  <img src="myimage.jpg" alt="Alternative Text">
```

Comment:
I would maybe expand `myimage.jpg` to include more of a path like `images/myimage.jpg`, because the reality is that media won't be hosted in the same directory as the view. It gives a little more context that that is a path to an image and not just the title. 

What is "Alternative Text" to someone who's just now learning about HTML and may not necessarily be aware of accessibility guidelines? Maybe something to the effect of "[Optional ]descriptive caption of the image [for screen readers]".

Reply:
I kept the examples the same as in the video, to minimize confusion if they want to reinforce the text with the video. But this is a good point. I think in the future it would be a good idea to rewrite all the examples in this lesson to be more realistic.

I added a bit to the explanatory text under the example about screen readers.